flang/test/Lower/intrinsic-procedures/AINT.f90

```diff
@@ -0,0 +1,9 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
```

Comment:
Should this go to nint?

Reply:
Wouldn't that be a bit inconsistent though?

Reply:
OK